### ISSUE TITLE
Artists: Participation options for artist entries regarding "group", "circle" and "label"

### ISSUE THREAD

**Issue by andreoda** [VocaDB/vocadb#1390]
Sometimes, people leave groups (teams), circles or labels. If we implement the option "past member", we can still maintain the fact that certain people belonged to a group/team.
Future confusion could be avoided through this as well. People might ask why a specific artist is mentioned in a specific entry.

Could be useful for the following issue:
![grafik](https://user-images.githubusercontent.com/52045122/120000365-0488a200-bfd3-11eb-9c51-cf2cc93e4521.png)
https://vocadb.net/Ar/84621
Specific members left the circle.

**Comment by andreoda** [comment]
**Update/New idea regarding this topic:**

Addition of (3) new options on artist entries in the "Groups and labels" field:
- (active) member
- past member/left
- guest member/artist

maybe also the option:
- Owner/Leader/Representative
(VGMdb has this option)

**Comment by FinnRG** [comment]
@Shiroizu @andreoda  This issue is now the most liked feature request on GH (tied with #1932), but there isn't a clear conclusion yet. There are both discussions of member status/rank (member, leader, guest), activity (active/past member) and time ranges.

I think we can ignore the time ranges for now. The status and activity is a bit trickier though, should the active/past switch be separate from the rank (guest/standard/leader)? Do we actually need "guest members"?

My questions boils down to: Should this be one checkbox (active member or not) and a dropdown (guest, standard, leader) or a single dropdown (with the values guest, past member, member, leader)?

**Comment by CatgirlFrostmoon** [comment]
"Guest members" doesn't really work - a person that collaborates once with the group isn't necessarily part of it. The status ranking is a bit strange in general, as I'm not sure how many actually have a leader, but it should be fine as long as that setting is optional and people don't try to attach it without strong evidence.

I'm fully for the activity split, though. Having a way of tracking past members is incredibly useful, and much better than the current info loss we face.

**Comment by FinnRG** [comment]
Does it maybe make sense to first implement the activity while we discuss the rank? I'm still not sure how activity and rank would interact with each other (does it make sense to have a past leader?) but we can figure that out later.

**Comment by CatgirlFrostmoon** [comment]
Yeah, I think activity first is a good idea! We can make a separate issue for the rank later.
